spring-cloud-kubernetes-test-support/src/main/java/org/springframework/cloud/kubernetes/integration/tests/commons/native_client/Util.java

```diff
@@ -132,6 +132,9 @@ public void createAndWait(String namespace, String name, V1Deployment deployment
 			}
 		}
 		catch (Exception e) {
+			if (e instanceof ApiException apiException) {
+				System.out.println(apiException.getResponseBody());
```

Comment:
Do we need this?

Reply:
well, I'd say yes. This can be printed via a Logger, but since `kubectl logs` takes stdout, it does not really matter. Especially since we use this for tests only. 

This `ApiException` is a bit weird, imho. If you do `apiException.printStackTrace()` - you will get a stack trace, but its going to be almost meaningless, the "real" issue is under `apiException.getResponseBody` - that is why I will slowly change this everywhere in our tests.